### ISSUE TITLE
Mailboxes, Domain etc. aren't displayed in Browser

### ISSUE THREAD

**Issue by Shumuu** [mailcow/mailcow-dockerized#1500]
Hey guys,

I installed mailcow-dockerized and configured mailboxes, domains etc. They are working fine, but I can't see them anymore in the GUI.

If I try to add a new mailbox, the domain is prechosen, but under mailbox quota it says I have to chose a domain.

Where did I go wrong ? What can I check ? To me it looks like the GUI has no connection to the database.

**Comment by andryyy** [comment]
Hi,

This does not happen on default installations, can you post `git diff origin/master`, please? Do you block JS in your browser?

**Comment by Shumuu** [comment]
Hey andryyy,

JS is allowed (chrome), if I type "git diff origin/master" nothing happens. I am in the mailcow-dockerized directory.

edit: I can see the logs under Debugging 

edit 2; I am running the update.sh - will report back as soon as it is finished. Should've run it before making a post probably ..

edit 3: Nope, still not working

**Comment by andryyy** [comment]
Cannot confirm or reproduce this on any machine.

Need more infos about the setup. Do you use a reverse proxy for example?

**Comment by Shumuu** [comment]
Soooo ... I had this issue for days now ...

I just switched the Language from German to English ... and now everything is there ...

I did the following.

I updated with the update script - still not working.
I resettet the admin password with the script - still not working.
I logged into the database and updated the admin password - still not working.
I changed the GUI language from German to English - he reloaded and everything is there, I can add mailboxes etc. No idea what that is, seriously. Can I provide any logs that might shed some light on this weird issue or do we just forget that it ever happened ?
